<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SonWidget1</class>
 <widget class="QWidget" name="SonWidget1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="startBotton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>467</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color:rgb(255, 255, 255);
	border-radius:5px;
	border:1px solid;
	border-color:rgb(172, 172, 172);
}
QPushButton:hover
{
	background-color:rgb(255, 255, 255);
	border:1px solid;
	border-color:rgb(208, 208, 208);
	border-radius:5px;
}</string>
   </property>
   <property name="text">
    <string>启动</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>241</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文隶书</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>467</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color:rgb(255, 255, 255);
	border-radius:5px;
	border:1px solid;
	border-color:rgb(172, 172, 172);
}
QPushButton:hover
{
	background-color:rgb(255, 255, 255);
	border:1px solid;
	border-color:rgb(208, 208, 208);
	border-radius:5px;
}</string>
   </property>
   <property name="text">
    <string>复原</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文彩云</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>300</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>期末要挂哪一科呢？</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>35</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color:rgb(255, 255, 255);
	border-radius:5px;
	border:1px solid;
	border-color:rgb(172, 172, 172);
}
QPushButton:hover
{
	background-color:rgb(255, 255, 255);
	border:1px solid;
	border-color:rgb(208, 208, 208);
	border-radius:5px;
}</string>
   </property>
   <property name="text">
    <string>编辑转盘</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
